.github/workflows/recreate-runner.yml

```diff
@@ -0,0 +1,22 @@
+name: Recreate Linux self-hosted runner on schedule
+
+# on:
+#   workflow_dispatch:
+#   schedule:
+#     # Once each 24 hours, at 1 during the night
+#     - cron: "0 0 1 1/1 *"
+
+on: pull_request
```

Comment:
This won't work because it won't have the secret. Make it `push` if this file is changed.

Reply:
I opened the PR as a branch from this repo, not from my fork, so it's fine I have the necessary permissions for my tests.